Examples/Neutral_density_calculation.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        # Print the time_step so the impatients know how much is left to do
</pre><p>But how do the impatients know how how many time steps are there? perhaps <code>print(100*time_step/T, "% done")</code> or something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>